README.md

```diff
@@ -92,6 +103,11 @@ Flight data and rocket parameters used in this comparison were kindly provided b
 |   NDRT launch vehicle   | Apogee time (s)        |    16.77  |    17.10  |   **-1.90 %**  |
 |   NDRT launch vehicle   | Maximum velocity (m/s) |   172.86  |   168.95  |   **2.31 %**   |
 
+RocketPy also displays several graphs for an better analisys, such as the one shown below.
+
+![Diagram](docs\static\rocketpy_example_trajectory.svg)
+
+<br/>
```

Comment:
Is the validation section the best place to add this image? How would it look just after the intro or perhaps in the end of the Running Your First Simulation section?

And it seems that you have not committed the image. Thus, the current preview for the ReadMe shows a missing image error.

![image](https://user-images.githubusercontent.com/572483/141031159-0478a6fb-77ec-4e63-8a4b-80a66e5863d0.png)

Let me know if you need any help with this.

Reply:
I think the image is not showing because it was not in my original pull request. But I made reference to the file you suggested previously. If, when commited, the image is still missing, I can adress that quickly. What do you think?

Reply:
I attempted to fix it before merging. Feel free to take a look at the commit to see what I did. I uploaded the image to the docs/static folder and commited it. Hope it works!